### ISSUE TITLE
Update on Current State of Clingo

### ISSUE THREAD

**Issue by stephanzwicknagl** [glaserL/viasp#82]
Hi @glaserL, 

a lot has happened to viASP since you last worked on it and I want to give you a little update. I also have a request in regards to publishing.

I recently finished my master's thesis on the project where I added features and made it easier to use overall. At this stage, Javier and I decided to move the repository (my fork of viasp) to the [potassco organization](https://github.com/potassco/viasp). You can try out the current version on binder: https://mybinder.org/v2/gh/potassco/viasp/main?filepath=examples%2FIntroduction%20to%20viASP.ipynb.

Now we would like to publish the current version to pypi. Since you are the owner of the three viasp packages (viasp, viasp-dash, viasp-backend) on pypi.org, there are a couple of options on how to update those from the new repository location:

- You add [me](https://pypi.org/user/stephanzwicknagl/) as a collaborator (owner) on the pypi project page, so I can manage the publishing (this option would be the least involved for you)
- You send me the API tokens, so that I can define the secrets in the github repo
- I ask Javier to give you the permissions to administer the potassco repo and you set the API secrets yourself 

You can choose any of these three ways and I will let you know any infos you need to finish it.

I hope to hear back from you soon!

Best,
Stephan

**Comment by glaserL** [comment]
Dear Stephan, 

first of all congratulation to finishing your thesis!

I don't really have a preference I think Javier and I talked about moving the ownership anyhow at some point in time but never did it. Would you also prefer to move the ownership?

And will there be a presentation on the new status? I'd like to see what has happened!

best regards
Luis

**Comment by stephanzwicknagl** [comment]
Thank you for the kind words! Unfortunately, the presentation already happened a couple weeks ago.

Yes, I think it would be best to move ownership of the pypi packages. I think you can add me as owner, that way you will still retain ownership, too. You can do this by adding my pypi account (username: stephanzwicknagl) on your project pages.

Best,
Stephan

**Comment by stephanzwicknagl** [comment]
Hi @glaserL,

I wanted to follow up in case you missed the previous updates to the issue. From my side, we can proceed with adding my account as owner of the viASP packages on pypi.org.

**Comment by glaserL** [comment]
You are absolutely right I forgot, sorry! Added you just now for all three packages, did you get the invite?

**Comment by stephanzwicknagl** [comment]
Thank you, I've received it! You can expect me to publish the updated versions within the next week or so.
